Fun Commands/Mini-Games/Dyno-RPG/README.md

```diff
@@ -0,0 +1,56 @@
+# Dyno RPG
+Otherwise called **Avelon RPG**, Dyno RPG is a RPG Game made using Dyno CC and inspirted by the infection minigame and a few other custom commands. It is planned to be a community prject to be built into something everybody can enjoy. As such everyone is welcome to add their own commands to it as they desire.
+
+
+## Registration Commands
+This command is used to join the world of Avelon
+
+`register`: Grants the **Adenturer** role and enters you itno the forsaken world of Avelon
+
+## Attack Commands
+These commands can be used to attack other players, but be warned some come with *side effects*
+
+`blight`: Cast Blight on another Adventurer
+`fester`: Cast Fester on another Adventurer
+`plague`: Cast a plague infecting all Adventurers
+`stab`: Stab another Adventurer 
+
+## Medica Commands
+These commands can be used to heal status ailments
+
+`treat`: Cure another Adventurer of all ailments
+`cure`: Cure yourself of all ailments
+`raise`: Raise another Adventurer from the dead
+
+## Divine Commands
+These commands can only be used by the Divine Council *[aka serverMods]*
+
+`divinecure`: Cures all Adventurers of all ailments
+
+
+### Requirements
+For this to work you will need the following roles in your guild
```

Comment:
Might want to add a colon to the end of this? More of a style thing like in line 42

Reply:
done